pkg/supportbundle/analyze.go

```diff
@@ -10,7 +10,9 @@ import (
 func InjectDefaultAnalyzers(analyzers []*troubleshootv1beta2.Analyze) []*troubleshootv1beta2.Analyze {
 	analyzers = append(analyzers, getAPIReplicaAnalyzer())
 	analyzers = append(analyzers, getNoGvisorAnalyzer())
-	analyzers = append(analyzers, getIfMissingKubernetesVersionAnalyzer(analyzers))
+	if a := getIfMissingKubernetesVersionAnalyzer(analyzers); a != nil {
+		analyzers = append(analyzers, a)
```

Comment:
do you think we should also check for nil here?

https://github.com/replicatedhq/troubleshoot/blob/cb5ddf752f30b68a6855476aa0ec3a889b1552cd/pkg/analyze/analyzer.go#L78

Reply:
i'll pr